### PR TITLE
refactor(no-drivable-lane): use getOrDeclareParameter

### PR DESCRIPTION
**Pull request by TakaHoribe**
## Description

use `tier4_autoware_utils::getOrDeclareParameter` for the ros parameter to prevent duplicate parameter declaration.

## Tests performed

Run psim

## Effects on system behavior

None

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by takayuki5168 — APPROVED** [review]

File: planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp [hunk 1]
```diff
@@ -14,17 +14,20 @@
...
 {
+using tier4_autoware_utils::getOrDeclareParameter;
 
 NoDrivableLaneModuleManager::NoDrivableLaneModuleManager(rclcpp::Node & node)
 : SceneModuleManagerInterface(node, getModuleName())
 {
   const std::string ns(getModuleName());
-  planner_param_.stop_margin = node.declare_parameter(ns + ".stop_margin", 1.5);
-  planner_param_.print_debug_info = node.declare_parameter(ns + ".print_debug_info", false);
+  planner_param_.stop_margin = getOrDeclareParameter<double>(node, ns + ".stop_margin");
```
Comment (TakaHoribe) [inline planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp:29]:
```suggestion
  planner_param_.stop_margin = getOrDeclareParameter<double>(node, ns + ".stop_margin", 1.5);
```

File: planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp [hunk 2]
```diff
@@ -14,17 +14,20 @@
...
+  planner_param_.print_debug_info = getOrDeclareParameter<bool>(node, ns + ".print_debug_info");
```
Comment (TakaHoribe) [inline planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp:30]:
```suggestion
  planner_param_.print_debug_info = getOrDeclareParameter<bool>(node, ns + ".print_debug_info, false");
```

File: planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp [hunk 3]
```diff
@@ -14,17 +14,20 @@
...
+using tier4_autoware_utils::getOrDeclareParameter;
 
 NoDrivableLaneModuleManager::NoDrivableLaneModuleManager(rclcpp::Node & node)
 : SceneModuleManagerInterface(node, getModuleName())
 {
   const std::string ns(getModuleName());
-  planner_param_.stop_margin = node.declare_parameter(ns + ".stop_margin", 1.5);
-  planner_param_.print_debug_info = node.declare_parameter(ns + ".print_debug_info", false);
+  planner_param_.stop_margin = getOrDeclareParameter<double>(node, ns + ".stop_margin", 1.5);
+  planner_param_.print_debug_info = getOrDeclareParameter<bool>(node, ns + ".print_debug_info, false");
```
Comment (TakaHoribe) [inline planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp:30]:
```suggestion
  planner_param_.print_debug_info = getOrDeclareParameter<bool>(node, ns + ".print_debug_info", false);
```

File: planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp [hunk 4]
```diff
@@ -14,17 +14,20 @@
...
 {
+using tier4_autoware_utils::getOrDeclareParameter;
 
 NoDrivableLaneModuleManager::NoDrivableLaneModuleManager(rclcpp::Node & node)
 : SceneModuleManagerInterface(node, getModuleName())
 {
   const std::string ns(getModuleName());
-  planner_param_.stop_margin = node.declare_parameter(ns + ".stop_margin", 1.5);
-  planner_param_.print_debug_info = node.declare_parameter(ns + ".print_debug_info", false);
+  planner_param_.stop_margin = getOrDeclareParameter<double>(node, ns + ".stop_margin", 1.5);
```
Comment (satoshi-ota) [inline planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp:29]:
```suggestion
  planner_param_.stop_margin = getOrDeclareParameter<double>(node, ns + ".stop_margin");
```

File: planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp [hunk 5]
```diff
@@ -14,17 +14,20 @@
...
+  planner_param_.print_debug_info = getOrDeclareParameter<bool>(node, ns + ".print_debug_info", false);
```
Comment (satoshi-ota) [inline planning/behavior_velocity_no_drivable_lane_module/src/manager.cpp:30]:
```suggestion
  planner_param_.print_debug_info = getOrDeclareParameter<bool>(node, ns + ".print_debug_info");
```

**Comment by ahmeddesokyebrahim** [comment]
Thank you @TakaHoribe san for this PR.
Actually I am trying to checkout this branch to test it in one of `no_drivable_lane` scenarios and I am not able to compile.
As well, there is some required gityub checks failing.
Can you please rebase and have a look to these checks ?
Thanks again for your efforts.

**Comment by TakaHoribe** [comment]
@ahmeddesokyebrahim Thank you for your review. I fixed the PR and also added a missing parameter in the node test. It should be fine I hope. Please check after the CI passes.

**Comment by TakaHoribe** [comment]
@ahmeddesokyebrahim Oops. Sorry, since I enabled an auto-merge, now it is merged!

**Pull request merged**